docs/Fabulous.XamarinForms/custom-renderer.md

```diff
@@ -0,0 +1,112 @@
+{% include_relative _header.md %}
+
+{% include_relative contents-views.md %}
+
+## Views: Custom Renderer / ViewElements
+
+Xamarin.Forms user interfaces are rendered using the native controls of the target platform, allowing Xamarin.Forms applications to retain the appropriate look and feel for each platform. Developers can implement their own custom Renderer classes to customize the appearance and/or behavior of a control.
+
+### Customized ViewElements
```

Comment:
I don't think the goal of those "customized ViewElements" is really clear. The TestLabel example is a technical one rather than a real-world use case.
Maybe it would be clearer to rephrase like:
>If an existing control does not provide the feature you want, you can extend it through a custom control.
>To do that, you can do this and that...

I just remembered a better example from FabulousContacts: https://github.com/TimLariviere/FabulousContacts/blob/master/FabulousContacts/Controls/UnderlinedLabel.fs

I created this UnderlinedLabel control because Xamarin.Forms does not provide underline option on Label while the native platforms support it (with the appropriate CustomRenderer)

Reply:
Also having a custom control without custom renderers is not really usual. Custom controls usually need custom renderers to apply the new features (like the underlining).
Though you can have just custom renderers without having to change the existing XF control.

Reply:
Don't hesitate to just redirect to the Xamarin.Forms documentation rather than explaining it yourself for all the Xamarin.Forms specific bits.
In addition to that, you can also point to one of our sample files (even FabulousContacts) for an example of "How it's done in F#".